pkg/splunk/client/minioclient.go

```diff
@@ -145,5 +154,8 @@ func (client *MinioClient) GetInitContainerImage() string {
 
 // GetInitContainerCmd returns the init container command on a per app source basis to be used by the initContainer
 func (client *MinioClient) GetInitContainerCmd(endpoint string, bucket string, path string, appSrcName string, appMnt string) []string {
-	return ([]string{fmt.Sprintf("--endpoint-url=%s", endpoint), "s3", "sync", fmt.Sprintf("s3://%s/%s", bucket, path), fmt.Sprintf("%s/%s", appMnt, appSrcName)})
+	s3AppSrcPath := filepath.Join(bucket, path) + "/"
+	podSyncPath := filepath.Join(appMnt, appSrcName) + "/"
+
+	return ([]string{fmt.Sprintf("--endpoint-url=%s", endpoint), "s3", "sync", fmt.Sprintf("s3://%s", s3AppSrcPath), podSyncPath})
```

Comment:
Just for my understanding, does this work for both minIO and GCP?

Reply:
Ya both minio and gcp endpoints can be used.